app/networkapi/news/models.py

```diff
@@ -1,5 +1,45 @@
 from django.db import models
 
+from networkapi.utility.images import get_image_upload_path
+
+
+def get_news_glyph_upload_path(instance, filename):
+    return get_image_upload_path(
+        app_name='news',
+        prop_name='headline',
+        instance=instance,
+        current_filename=filename
+    )
+
+
+class Topic(models.Model):
+    """
+    News Topics/Categories
+    """
+    name = models.CharField(max_length=300)
+
 
 class News(models.Model):
```

Comment:
can you update the class so that it's not pluralized as "Newss" in the admin console?